src/reports.cpp

```diff
@@ -394,7 +395,7 @@ REPORT_GENERATOR(unit_alignment, rc)
 	const map_location& hex = mouseover_hex.valid() ? mouseover_hex : displayed_unit_hex;
 	return unit_alignment(rc, u, hex);
 }
-REPORT_GENERATOR(selected_unit_alignment, rc)
+REPORT_GENERATOR(selected_unit, rc)
```

Comment:
???

This doesn't look right? Why would you change the report name?

Reply:
As @CelticMinstrel says, this looks like a bug. These macros turn into ThemeWML tag names, so this makes it expect a `[selected_unit]` instead of a `[selected_unit_alignment]` in the theme.

I think anything that starts `[selected_` isn't exercised by the built-in themes, but can be used by jostephd's two-sidebar theme (and probably others).

Reply:
Seems likely this was just an overzealous search-and-replace error.

Reply:
Ah, yeah, I was actually un-renaming another report after noticing changing *that* report generator name caused the unit alignment to no longer be displayed on the sidebar, and somehow this one got caught up in that.

Reply:
Fixed.